src/utilities/useI18n.ts

```diff
@@ -0,0 +1,10 @@
+export function useI18n(): Record<string, any> {
+  return {
+    // A map of (machine-readable) terms (e.g. fields that come from API data) to (human-readable) labels to used in the UI.
+    termLabels: {
+      mtls: 'mTLS',
+    },
+  }
+}
+
+export type I18n = ReturnType<typeof useI18n>
```

Comment:
Just a small note here (I noticed this was amended since I last looked at this PR)

Preferably this would return a pseudo-standard `t('json.path.term')` function instead of just a JSON blob.

`t('json.path.term')` to me says "this is text/copy" rather than just `json.path.term` which to me says "this is data"

We make and document these decisions in the MADR though

Reply:
Until we’ll decide on how to proceed with matters of internationalization, I’ll leave this as-is.

Reply:
Yep no prob